lib/math.js

```diff
@@ -0,0 +1,38 @@
+function total(nums){
+  return nums.reduce(function(sum, num){
+    return sum + num;
+  });
+};
+
+function mean(nums){
+  return total(nums) / nums.length
+};
+
+function median(nums){
+  return mean(nums.slice(Math.ceil(nums.length/2)-1, (nums.length/2)+1))
+};
+
+function mode(nums){
+  return nums.reduce(function(map, num){
+    let now = map.numbers[num] = (map.numbers[num] || 0) + 1
+    if (now > map.mostFrequent){
```

Comment:
I'd work to be more consistent with semicolons after expressions....for example, line 17 should be followed by `;`